pibooth/plugins/view_plugin.py

```diff
@@ -162,8 +162,9 @@ def state_print_enter(self, cfg, app, win):
     @pibooth.hookimpl
     def state_print_validate(self, app, win, events):
         printed = app.find_print_event(events)
-        self.forgotten = app.find_capture_event(events)
-        if self.print_view_timer.is_timeout() or printed or self.forgotten:
+        has_capture_event = app.find_capture_event(events)
+        self.forgotten = has_capture_event and app.can_forget
+        if self.print_view_timer.is_timeout() or printed or has_capture_event:
```

Comment:
I would just rename the variable
```suggestion
        capture_pressed = app.find_capture_event(events)
        self.forgotten = capture_pressed and app.can_forget
        if self.print_view_timer.is_timeout() or printed or capture_pressed:
```